enterprise/server/remote_execution/workspace/workspace.go

```diff
@@ -187,10 +183,14 @@ func (ws *Workspace) AddCIRunner(ctx context.Context) error {
 	if exists {
 		return nil
 	}
+
+	// Make sure that we load the ci runner from embedded bundle instead of runfiles
+	// because we do not deploy MacOS Executor with runfiles.
```

Comment:
I think this comment is inaccurate since we haven't yet made the change to only load the CI runner from `embed.FS`.

Currently, `ci_runner_bundle` will load from either runfiles or `embed.FS` depending on what's available.

```suggestion
```

Reply:
I would say that it's not a surefire way to achieve this. However, because we are exporting `ci_runner_bundle.BundlePrefix` and using the exported value, I expect `fileresolver.go` to always look up the file inside the embedded bundle.

A better way to achieve this would be to export the embedded FS and use it directly without going through fileresolver. WDYT if we take that route instead?

Reply:
After tinkering with that route, it seems like we have call sites that expect this bundle to act as either embedded fs or runfiles fs. Refactoring this bundle into 2 separate FS could be time-consuming so let's get this merged for now.

Reply:
Looks like the test is failing to find the CI runner binary: https://buildbuddy.buildbuddy.io/invocation/8a482fc8-52c7-438f-9574-33864a71babb?target=%2F%2Fenterprise%2Fserver%2Ftest%2Fintegration%2Fworkflow%3Aworkflow_test&targetStatus=6

can you double check that this PR is a clean revert?